themes/bootstrap3/less/components/search.less

```diff
@@ -427,6 +427,10 @@ body.rtl {
 .combined-search-container.grid {
   margin-left: 1rem;
   margin-right: 1rem;
+
+  .combined-list {
+    overflow-wrap: anywhere;
```

Comment:
> I also agree that the overflow scrollbar is not good. I'd prefer hidden but there's more we can do, I think. Since this happens when long words come into play, we can allow for more breaks in words:
> 
> ```css
> .combined-list {
>     word-break: break-word;
>     hyphens: auto;
> }
> ```
> 
> We can also do [what we do locally at Villanova](https://library.villanova.edu/Find/Combined/Results?type=&lookfor=behold&limit=20&sort=relevance) and make the container on the combined screen full width.

@crhallberg Yes this is so much better than a scrollbar, thanks for the idea.  MDN said word-break: break-word is deprecated so I used its suggested replacement.  Overflow-wrap: break-word didn't work either but :anywhere did.  The hyphens rule didn't affect it, but actually given that what is wrapping (mostly) are URLs, I'm happier not hyphenating them as I think that confuses things further.

Reply:
Oh! I hadn't realized that! Thank you for the heads up!